Core/NoOpRunner.Core.Tests/ShapeFactoryTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NoOpRunner.Core.Enums;
+using Xunit;
+using Shouldly;
+using NoOpRunner.Core.Shapes.ShapeFactories;
+using NoOpRunner.Core.Shapes.EntityShapes;
+
+namespace NoOpRunner.Core.Tests
+{
+    public class ShapeFactoryTests
+    {
+        [Fact]
+        public void GetShape_WhenValidShapeAndCoordinatesGiven_ShouldReturnAppropriateShape()
+        {
+            // Arrange
+            var expected = new DamageCrystal(5, 5);
+            ShapeFactory shapeFactory = new ShapeFactory();
+            var actual = shapeFactory.GetShape(Shape.DamageCrystal, 5, 5);
```

Comment:
**[Nitpick]** No `// Act` mark, it's nice to have all the three steps, easier to comprehend the test at a glance. Same for a few other places

Reply:
Fixed